### ISSUE TITLE
Namespaces breaks library

### ISSUE THREAD

**Issue by absfarah** [chriskacerguis/codeigniter-restserver#747]
The new namespace changes breaks things for me. Same for someone else on [StackOverFlow](http://stackoverflow.com/questions/41384909/class-rest-controller-not-found-with-codeigniter-and-rest-api-service/41386921#41386921).

I download the library on pretty much a vanilla CI 3.0 download and I get the error:

> Class 'REST_Controller' not found

After removing the namespaces changes, everything works fine. So not sure what the exact issue is. Not great with namespaces so I can help further, apologies.

**Comment by chriskacerguis** [comment]
@absfarah looks like you have not initialized it correctly.  We'll update the docs accordingly.

**Comment by vinnieolle** [comment]
I have a problem that might be related. I can only get the server to work if I comment out the following lines from REST_Controller.php
```
namespace Rest_server\Libraries;

use Exception;
use stdClass;
use Rest_server\Libraries\Format;
```
and this following from Format.php
`namespace Rest_server\Libraries;`
Otherwise I get this:
> Fatal error: Class 'REST_Controller' not found in C:\xampp\htdocs\codeigniter\application\controllers\api\Example.php on line 21
> A PHP Error was encountered
> Severity: Error
> Message: Class 'REST_Controller' not found
> Filename: api/Example.php
> Line Number: 21
> Backtrace

I'm very, very new to this so sorry if I am overlooking something simple, but I couldn't find the answer anywhere and have tried starting from the beginning a few times with no luck.

**Comment by atukilham** [comment]
i have some problem +1

**Comment by nasznjoka** [comment]
What could be the cause of this? Cause I tried to load the Format.php file in autoload it said the file can not be found! I tried to include_once() the REST_controller file in the example controller it says the file can not be found! Even after commenting out those namespace staffs

**Comment by chriskacerguis** [comment]
The issue is with namespaces (it's not actually an issue, just something didn't get updated in the PR.  I should have a fix shortly.

**Comment by chriskacerguis** [comment]
I just updated the code with the fixes (I have also included CI 3.1.2 to make life easier for all)

https://github.com/chriskacerguis/codeigniter-restserver/commit/524718b31e31e0bd6c4b26a6770a0f9395f7f2c1

**Comment by lweb20** [comment]
using Restserver\Libraries;
